frontend/simple/views/components/Voting/Voting.vue

```diff
@@ -130,36 +153,130 @@ export default {
   },
   data () {
     return {
-      closeProposalState: {
+      ephemeral: {
```

Comment:
This state is only related to "close" view of this componet, so IMO we should keep that. Something like "closeEphemeral" ?

Reply:
@sandrina-p `ephemeral` comes from [our data guidelines](https://github.com/okTurtles/group-income-simple/blob/master/docs/Style-Guide.md#how-to-organize-data-)

Reply:
I also expect some other ephemeral states here, eg. the thing what happens after reaching the threshold we talked about. we'll see there :)